pom.xml

```diff
@@ -15,6 +15,7 @@
         <java.version>17</java.version>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
+        <buildDirectory>${project.basedir}/target</buildDirectory>
```

Comment:
This is the default build output directory and is referenced later.
However, providing it as a property allows us to override it at the command line to allow for parallel builds.